applications/dashboard/views/settings/ban.php

```diff
@@ -17,14 +17,14 @@
             echo $this->Form->label('Value or Pattern', 'BanValue');
             echo $this->Form->textBox('BanValue');
             ?>
-            <span>Use asterisks for wildcards, e.g. &lsquo;*@hotmail.com&rsquo;</span>
+            <span><?php echo t('Use asterisks for wildcards, e.g. &lsquo;*@hotmail.com&rsquo;'); ?></span>
```

Comment:
`t('Use asterisks for wildcards', 'Use asterisks for wildcards, e.g. &lsquo;*@hotmail.com&rsquo;')`